apps/pie-storybook/stories/pie-link.stories.ts

```diff
@@ -98,7 +104,9 @@ const Template = ({
             rel="${rel || nothing}"
             ?isBold="${isBold}"
             ?isStandalone="${isStandalone}">
+            ${iconPlacement === 'leading' ? html`<icon-plus-circle slot="icon-leading"></icon-plus-circle>` : ''}
```

Comment:
You might want to use `nothing` as opposed to an empty string being rendered in the markup:
```javascript
${iconPlacement === 'leading' ? html`<icon-plus-circle slot="icon-leading"></icon-plus-circle>` : nothing}
```